docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,247 @@
+
+2. In the **Callback URLs** section, supply the following URLs, replacing `{ngrok_id}` with your ngrok ID from the preceding section.
+
+| Callback | URL value |
+|:---------|:----|
+| Auth Callback | `https://{ngrok_id}.ngrok.io/api/auth` |
+| Load Callback | `https://{ngrok_id}.ngrok.io/api/load` |
+| Uninstall Callback | `https://{ngrok_id}.ngrok.io/api/uninstall` |
+
+3. In the **OAuth scopes** section, add the following scopes:
+
+| UI Name | Permission | Parameter |
+|:--------|:-----------|:----------|
+| Orders | modify | `store_v2_orders` |
+| Order Transactions | modify | `store_v2_transactions` |
+| Products | modify | `store_v2_products` |
+| Customers | modify | `store_v2_customers` |
+| Content | modify | `store_v2_content` |
+| Carts | modify | `store_cart` |
+| Channel Listings | modify | `store_channel_listings` |
+| Channel Settings | modify | `store_channel_settings` |
+| Information & Settings | modify | `store_v2_information` |
+| Sites & Routes | read-only | `store_sites_read_only` |
+| Storefront API Tokens | generate tokens | `store_storefront_api` |
+
+4. On the **App Supported Features** tab, indicate that this is a multi-storefront enabled app.
+
+5. Click **Update & Close**, then click **Confirm Update** to acknowledge the OAuth scopes you configured.
+
+Further steps in this guide require access to the Dev Portal, so keep it handy.
+
+## Configure Stripe
+
+We've designed this integration to use Stripe Connect so that your app can use one connection to submit payments to multiple merchants' previously configured Stripe accounts. See [Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing) for merchant configuration instructions.
+
+The respective architectures of the Stripe and BigCommerce APIs on which this app relies require the app to make charges using the **merchant-specific Stripe account's public key** and the **app-specific Stripe account's secret key**. 
+
+This arrangement provides the following benefits:
+* The merchant can change the Stripe account they use without breaking the integration.
+* The merchant's own Stripe account stores all shopper payment history, subscription and one-time purchase alike. 
+* It limits the app's PCI compliance burden. 
+
+In development, you'll work with two Stripe accounts; one that simulates a merchant's pre-existing account, and another that you configure as the app's dedicated Stripe Connect-enabled account.
+
+<!-- theme: info -->
+> #### Multi-tenant mechanics
+> During installation, the app UI prompts the merchant to enter their Stripe account's keys so that the app can initiate an OAuth grant code authorization flow that links the merchant's Stripe account to the app's Stripe Connect-enabled account. 
+
+### Create Stripe accounts
+
```

Comment:
I think we should include a brief intro to explain why we are creating multiple Stripe accounts and what each account is used for (merchant stripe account, app stripe account. Consider displaying those accounts in a table so the users has all the info in one place.